webapp/src/containers/Rhs/rhs.scss

```
@@ -81,6 +81,12 @@
     box-shadow: 0 8px 24px rgba(0, 0, 0, 0.15);
   }
 
+  .rhs-menu-popover {
+    right: 12px;
+    width: 310px;
+    padding: 0px;
```

```suggestion
    padding: 0;
```
no need to specify unit for `0`